WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -105,6 +105,10 @@ $venue-map-wrapper-height: 400px;
   }
 }
 
+#map-wrapper {
+  height: 600px;
```

Comment:
As we've discussed before, I don't believe any fixed number is going to work well here, I'd like to have the map take up all the available space instead. Did you look into that?